packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2036,7 +2134,7 @@ export class ContainerRuntime
 					this.idCompressor !== undefined,
 					0x67b /* IdCompressor should be defined if enabled */,
 				);
-				return this.applyStashedIdAllocationOp(contents as IdCreationRangeWithStashedState);
+				return this.applyStashedIdAllocationOp(opContents.contents);
```

Comment:
I wonder how many `as`'s we can kick out this week 😇

Reply:
That is the metric, right?